packages/fields/src/Controller.js

```diff
@@ -40,8 +40,8 @@ export default class FieldController {
    * passed to this.serialize() for you
    * @return boolean
    */
-  hasChanged = async (initialData, currentData) =>
```

Comment:
This function should _not_ be `async`, it is consume _synchronously_ here:

https://github.com/keystonejs/keystone-5/blob/fix-has-changed/packages/app-admin-ui/client/pages/Item/index.js#L134-L137